tests/hardhat/Fork/reduceReservesTest.ts

```diff
@@ -110,8 +116,8 @@ if (FORK_TESTNET || FORK_MAINNET) {
 
       await comptroller.setMarketSupplyCaps([vUSDD.address], [convertToUnit(1, 50)]);
       await comptroller.setMarketBorrowCaps([vUSDD.address], [convertToUnit(1, 50)]);
-
-      await protocolShareReserve.connect(impersonatedTimelock).setPoolRegistry(POOL_REGISTRY);
+      await protocolShareReserve.connect(impersonatedProtocolSeizeAdmin).setPoolRegistry(POOL_REGISTRY);
+      await vUSDD.connect(impersonatedTimelock).setProtocolShareReserve(PROTOCOL_SHARE_RESERVE);
```

Comment:
I don't think we should do this in the test. Currently, the protocolShareReserve address is wrong in the markets of the pool, so it's if this test fails because of that. If we set the protocolShareReserve value here we would hide the current issue and the potential issues in the future

Reply:
Will update after testnet updates